ecs/amazon-es.conf

```diff
@@ -0,0 +1,21 @@
+[SERVICE]
+    Grace 30
+
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[FILTER]
+    Name record_modifier
+    Match *
+    Record ecs_cluster ${ECS_CLUSTER}
+    Record ecs_task_arn ${ECS_TASK_ARN}
+    Record ecs_task_definition ${ECS_TASK_DEFINITION}
+
+[OUTPUT]
+    Name  es
+    Match *
+    AWS_Region ${ES_REGION}
+    AWS_STS_Endpoint ${ENDPOINT}
```

Comment:
the endpoint is for elasticsearch. Its the `Host` field. Also this config will not actually work to send logs to Amazon ES. Please see and copy this: https://docs.fluentbit.io/manual/pipeline/outputs/elasticsearch##aws-es

Reply:
also, from my doc, you are missing the INDEX and TYPE env vars

Reply:
also, I just updated the doc, the endpoint should be `ES_ENDPOINT` for the env var

Reply:
```
    Index ${INDEX}
    Type  ${TYPE}
```
are these enough for INDEX and TYPE env vars?

Reply:
Yes